plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -9,21 +7,135 @@
 		width: 100%;
 	}
 
+	.settings-payment-providers__header-title {
+		font-size: 16px;
+		font-weight: 600;
+		line-height: 24px;
+	}
+	.settings-payment-providers__header-select-container {
+		.components-input-control__prefix {
+			min-width: fit-content;
+			padding-left: 12px;
+			color: $gray-700;
+		}
+	}
 
-	table.wc_gateways {
-		.other-payment-methods__button-text {
-			margin-right: 4px;
+	// Override styles for woocommerce list items.
+	.woocommerce-list__item {
+		&:hover {
+			background-color: $white;
+
+			.woocommerce-list__item-title {
+				color: $gray-900;
+			}
 		}
+	}
+	.woocommerce-list__item-inner {
```

Comment:
probably can be stacked under `.woocommerce-list` and using `&__` I may fix it in a bit

Reply:
done in https://github.com/woocommerce/woocommerce/pull/52030/commits/0444c8c78b346bc2b8061a88e4e8898d5f3357f7